web-console/src/dialogs/retention-dialog/retention-dialog.tsx

```diff
@@ -194,11 +181,21 @@ ORDER BY 1`,
       {datasource !== CLUSTER_DEFAULT_FAKE_DATASOURCE && (
         <>
           <Divider />
-          <FormGroup>
-            <p>
-              Cluster defaults (<a onClick={onEditDefaults}>edit</a>):
-            </p>
-            {defaultRules.map(renderDefaultRule)}
+          <FormGroup
+            label={
+              <>
+                Cluster defaults (<a onClick={onEditDefaults}>edit</a>)
+              </>
+            }
+          >
+            <p>The cluster default rules are applied if no datasource specific rule matches.</p>
```

Comment:
```suggestion
            <p>The cluster default rules are evaluated if none of the above rules match.</p>
```

We don't explicitly mention that the rules above are datasource specific rules. And we say those rules are evaluated, while this message says applies, which made me wonder what the difference was.